homeassistant/components/tesla_fleet/sensor.py

```diff
@@ -378,6 +378,14 @@ class TeslaFleetTimeEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         entity_registry_enabled_default=False,
     ),
+    SensorEntityDescription(
+        key="island_status",
+        entity_registry_enabled_default=True,
```

Comment:
True is the default value so you can just remove this line.

Reply:
Fixed in [a682d25](https://github.com/home-assistant/core/pull/126438/commits/a682d2504606bbb38b4dfe8fe27fea06a36a81ea).